portfolio/src/main/java/com/google/sps/servlets/DiscussionServlet.java

```diff
@@ -121,6 +121,9 @@ private String convertToJson(ArrayList<Post> data) {
         String replyTime = String.valueOf(cur.getProperty("replyTime"));
         String replyUserEmail = (String) cur.getProperty("userEmail");
 
+        Translation translationReplyContent = translate.translate(replyContent, Translate.TranslateOption.targetLanguage(languageCode));
```

Comment:
Try to follow line length style guide.
https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

a tip: import Translate.TranslateOption instead of just Translate

Reply:
Thanks!

Reply:
I did import TranslateOption at the top of the file though.

import com.google.cloud.translate.TranslateOptions;